www/include/views/graphs/exportData/ExportCSVServiceData.php

```diff
@@ -58,31 +58,59 @@ function get_error($str)
     get_error('need session id !');
 }
 
-isset($_GET["index"]) ? $index = htmlentities($_GET["index"], ENT_QUOTES, "UTF-8") : $index = null;
-isset($_POST["index"]) ? $index = htmlentities($_POST["index"], ENT_QUOTES, "UTF-8") : $index = $index;
+$index = filter_var(
+    $_GET['index'] ?? $_POST['index'] ?? null,
```

Comment:
```suggestion
    $_GET['index'] ?? $_POST['index'] ?? false,
```
using false may help you check if the variable of the not set or a string was given, as $index will be false in either cases

Reply:
are you sure, that a POST request is available on this param ? as the other variables do not seems to use POST

Reply:
It seemed to be the case when looking to the previous code

Reply:
Will this not look weird and "wrong" is I set this to `false`
We expect a int variable for this one, I would expect it then to be a integer or null

WDYT ?

Reply:
indeed, but could you check if it's consistent ;)

Reply:
Sure I'll add then a check on

- Is it defined (to not break the request) ?
- Is it an integer ? (to avoid string)